src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = getReadFromFile(fromFileName);
+        String report = getReportToFile(dataFromFile);
+        getWriteToFile(report,toFileName);
+    }
+
+    private String getReadFromFile(String fromFileName) {
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(fromFileName))) {
+            StringBuilder builder = new StringBuilder();
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                builder.append(value).append(" ");
```

Comment:
better use System.lineSeparator than " ", and either value should be constant (reused in generate report method)